images/bf_logo_white.svg

```diff
@@ -1,47 +1,47 @@
-<?xml version="1.0" encoding="utf-8"?>

```

Comment:
This is my change that slipped in when a file got converted from DOS to unix when pushing some fixes for code smells. Apologies.

Reply:
@mikeller No problem, how to solve?

Reply:
@ctzsnooze you can resolve by reverting the lines and push again otherwise other changes would be lost.